helm/methode-image-binary-mapper/values.yaml

```diff
@@ -8,8 +8,10 @@ replicaCount: 2
 image:
   repository: coco/methode-image-binary-mapper
   pullPolicy: IfNotPresent
-#resources:
-#  limits:
-#    memory: 256Mi
+resources:
+  requests:
+    memory: 360Mi
```

Comment:
Looks a bit low when I look at Grafana, the average memory usage is between 420-440Mi.

Reply:
That's right, I'll increase it.